addons/sale_project/static/src/views/components/project_template_dropdown.js

```diff
@@ -0,0 +1,16 @@
+import { patch } from "@web/core/utils/patch";
+import { ProjectTemplateDropdown } from "@project/views/components/project_template_dropdown";
+
+patch(ProjectTemplateDropdown.prototype, {
+    get readFields() {
+        const result = super.readFields;
+        result.push("allow_billable");
+        return result;
+    },
+
+    contextPreprocess(templateId) {
+        const result = super.contextPreprocess(...arguments);
+        result.is_template_billable = this.projectTemplates.find(item => item.id === templateId).allow_billable;
```

Comment:
should be better to create a related field in project.template.create.wizard instead of adding additional context.